src/components/Pagination/Pagination.vue

```diff
@@ -0,0 +1,315 @@
+<template>
+  <div
+    v-if="shouldRender"
+    class="pagination"
+  >
+    <div>
+      <p class="pagination__results">
+        {{ totalNumber }} result<span v-if="totalNumber !== 1">s</span>
+      </p>
+    </div>
+
+    <div class="pagination__paging-container">
+      <p class="pagination__text">
+        {{ paginationText }}
+      </p>
+      <button
+        :class="[
+          'pagination__btn',
+          { 'pagination__btn--disabled': page === 1 },
+        ]"
+        :disabled="page === 1"
+        @click="pageClick(1)"
+      >
+        <span class="sr-only">Go to first page</span>
+        <svg
+          width="16"
+          height="16"
+          viewBox="0 0 16 16"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
```

Comment:
We talked about why we weren't able to extract these SVGs into their own files in the AngularJS version - does that limitation apply to Vue too? If not, can we do that here?

Reply:
Created a component for the page icons. Put all the icons in there rather than 1 file each because the styles and props were the same for all of them and these are only used in one place. (and honestly might be going away...)